components/storage-sync/sql/create_schema.sql

```diff
@@ -0,0 +1,27 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-- This is a very simple schema.
+CREATE TABLE IF NOT EXISTS moz_extension_data (
+    /* The GUID is a hash of the extension ID - we explicitly do not store
+       the raw ID in the DB
+    */
+    guid TEXT PRIMARY KEY,
+    /* The JSON payload. not null because we prefer to delete the row than null it */
+    data LONGVARCHAR NOT NULL,
```

Comment:
SQLITE doesn't distinguish between LONGVARCHAR and TEXT (rules are https://www.sqlite.org/datatype3.html)